tools/DSharpPlus.Tools.ShardedEventHandlingGen/Program.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System.Globalization;
+using System.Reflection;
+using System.Text;
+using DSharpPlus;
+using Humanizer;
+
```

Comment:
is humanizer really needed here? also, extra newline

Reply:
I can remove it, added it for prototyping and i think `.Camelize` is the only method survived which is from it